etc/kayobe/environments/ci-multinode/stackhpc-ci.yml

```diff
@@ -48,5 +48,11 @@ stackhpc_repo_docker_ce_ubuntu_version: "{{ stackhpc_pulp_repo_docker_ce_ubuntu_
 stackhpc_docker_registry: "{{ stackhpc_repo_mirror_url | regex_replace('^https?://', '') }}"
 
 # Username and password of container registry.
-stackhpc_docker_registry_username: ""
-stackhpc_docker_registry_password: ""
+stackhpc_docker_registry_username: "stackhpc-kayobe-ci"
+stackhpc_docker_registry_password: !vault |
```

Comment:
Did you create a new vault password for this?

Reply:
I used the one you provided. However, I think there was some confusion at the start of this project regarding the password.

Reply:
I don't mean the registry password, but the ansible vault password used to encrypt it here